### PR TITLE
add maestro_cfg_test to TADA test suite

### PR DESCRIPTION
**Pull request by nick-enoent**
fix typos

### CODE REVIEW

File: maestro_cfg_test [hunk 1]
```diff
@@ -0,0 +1,369 @@
...
+NUM_COMPUTE = 8
+STORE_ROOT = "/store" # path inside container (aggs)
+CLUSTER_CONFIG = "etcd.yaml"
+CFG_PREFIX = "headnode"
+AGG_PORT = 10010
+SMPLR_PORT = 10002
+HEADNODE_PORT = 10104
+STORE_PATH = "/DATA15/orion"
+LDMS_CFG_FILE = "/opt/ovis/ldms_cfg.yaml"
+ETCD_FILE = "/opt/ovis/etcd.yaml"
```
Comment (narategithub) [inline maestro_cfg_test:64]: I think `STORE_PATH` should be based on `DB` (which could be specified via --data-root in the CLI). My system does not have "/DATA15/orion" directory.

Also, can `LDMS_CFG_FILE` and `ETCD_FILE` be in the `DB` directory? People running the test may not have write permission to `/opt/ovis` directory on the host, or their ovis installation (on the host, not container) may be in different path.

Comment (nick-enoent) [reply maestro_cfg_test:64]: Whoop, good catch on the STORE_PATH, and sure, good point. I'll change over LDMS_CFG_FILE and ETCD_FILE to be in DB directory as well.

**Comment by narategithub** [comment]
@nick-enoent did you prep a docker image with maestro dependencies? (e.g. etcd and yaml python package). If so, would you please also provide the change in the Dockerfile?

**Comment by nick-enoent** [comment]
@narategithub etcd is mounted from the host machine in /opt/ovis, and I was under the impression that PyYAML was packaged with python3, but I think I was mistaken. I'll add the relevant python packages and update the commit!

**Comment by narategithub** [comment]
@nick-enoent I C. Thanks!

**Comment by nick-enoent** [comment]
@narategithub Do we still use python2 for anything, or could I remove those installations from the Dockerfile?

**Comment by narategithub** [comment]
@nick-enoent I don't think we python2 anywhere. Let's remove it.

**Pull request merged**